tests/cc/test_queuestack_table.cc

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2020 Politecnico di Torino
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "BPF.h"
+
+#include "catch.hpp"
+
+#include <iostream>
+
```

Comment:
Thanks. Looks good! There are some spaces between `#include`s. Could you remove them?

Reply:
Sure, I've just re-pushed the code with no spaces

Reply:
Thanks!